### ISSUE TITLE
Sitecore specific tokens used in script body are resolved during deserialization.

### ISSUE THREAD

**Issue by alan-null** [SitecorePowerShell/Console#348]
**How to reproduce:**
1  Navigate to the script library: */sitecore/system/Modules/PowerShell/Script Library/Library/PowerShell Script*
2  Create new script
3  Enter following content into script body
``` powershell
$name
$id
$now
```
4  Serialize script
5  Remove script from database
6  Update Script library tree to load script into database from disk.

**Actual result:**
Script will be deserialized with following content
``` powershell
PowerShell Script
{F11EB250-B355-4D35-B2AD-489317F21AFA}
20150527Z
```

**Expected result:**
```
$name
$id
$now
```

**Comment by AdamNaj** [comment]
That's a great contribution. As we talked... this has hit us in the past, but no longer :)

**Comment by AdamNaj** [comment]
Merged by implementing Issue #350
